src/myframe.h

```diff
@@ -1036,10 +1036,8 @@ struct MyFrame : wxFrame {
     }
 
     void OnSearch(wxCommandEvent &ce) {
-        return search(ce.GetString());
-    }
-
-    void search(wxString searchstring) {

```

Comment:
Partially revert the change before because there is no need to split the function. Instead of trigger a new search in case case sensitivity is toggled (see above), just refresh the document.